File: particle-connect/android/src/main/kotlin/network/particle/connect_plugin/model/RpcUrl.java
package network.particle.connect_plugin.model;

import androidx.annotation.Keep;

import com.google.gson.annotations.SerializedName;
@Keep
public class RpcUrl {

    @SerializedName("evm_url")
    public String evmUrl;

    @SerializedName("sol_url")
    public String solUrl;
}

File: particle-auth/android/src/main/kotlin/network/particle/auth_flutter/bridge/model/RpcUrl.kt
package network.particle.auth_flutter.bridge.model

import com.google.gson.annotations.SerializedName

class RpcUrl {
    @SerializedName("evm_url")
    var evmUrl: String? = null

    @SerializedName("sol_url")
    var solUrl: String? = null
}